src/main/java/com/nike/cerberus/cli/EnvironmentConfigToArgsMapper.java

```diff
@@ -167,12 +167,12 @@ private static void addCommonStackArgs(EnvironmentConfig environmentConfig, List
     }
 
     private static void addTagArgs(EnvironmentConfig environmentConfig, List<String> args) {
-        args.add(TagParametersDelegate.COST_CENTER_LONG_ARG);
-        args.add(environmentConfig.getCostCenter());
-        args.add(TagParametersDelegate.OWNER_EMAIL_LONG_ARG);
-        args.add(environmentConfig.getOwnerEmail());
-        args.add(TagParametersDelegate.OWNER_GROUP_LONG_ARG);
-        args.add(environmentConfig.getOwnerGroup());
+//        args.add(TagParametersDelegate.COST_CENTER_LONG_ARG);
```

Comment:
I have another PR queued up after this one, where I work on class and the yaml to handle the new infrastructure and generic tags